lib/tokenizeCommandString.js

```diff
@@ -1,5 +1,26 @@
 /* global tokenizeCommandString:true */
 /* exported tokenizeCommandString */
 
-// we need to filter out non-args in case the command string was `/cycle `
-tokenizeCommandString = str => str.split(/\s+/).filter(arg => arg.trim().length > 0)
+tokenizeCommandString = commandStr => {
+  let inSingleQuotes = false
+  let inDoubleQuotes = false
+  const commandArray = commandStr.split('')
+  // replace argument-delimiters with '\n' which we'll use to split-on later
+  for (let i = 0; i < commandArray.length; ++i) {
+    if (commandArray[i] === "'" && !inDoubleQuotes) {
+      inSingleQuotes = !inSingleQuotes
+      commandArray[i] = '\n'
+    }
+    if (commandArray[i] === '"' && !inSingleQuotes) {
+      inDoubleQuotes = !inDoubleQuotes
+      commandArray[i] = '\n'
+    }
+    if (commandArray[i] === ' ' && !inSingleQuotes && !inDoubleQuotes) {
+      commandArray[i] = '\n'
+    }
+  }
+  return commandArray
+    .join('')                        // turn array back into string
+    .split('\n')                     // split the arguments into an array
+    .filter(arg => arg.length > 0)   // empty arguments aren't arguments
+}
```

Comment:
This algorithm doesn't provide a way to escape the chosen delimiter (single or double quote).

![mpeiiu3tow](https://cloud.githubusercontent.com/assets/189699/16606010/7aeb5546-42f1-11e6-9b03-77492837ded9.gif)

This may seem like an edge case, but it means that there is no way to give an answer with both types of quotes in it like:
```
My favorite thing about Jeffrey is how he always tells me, "You've got this!"
```